pacman/model/graphs/graph.py

```diff
@@ -155,48 +179,36 @@ def add_edge(self, edge, outgoing_edge_partition_name):
 
         if edge.pre_vertex.label not in self._vertex_by_label:
             raise PacmanInvalidParameterException(
-                "edge", edge.pre_vertex, "pre-vertex must be known in graph")
+                "edge", str(edge.pre_vertex),
```

Comment:
Capital letter here and a few lines on from here.

Reply:
fixed